app/components/Settings/scheduler/scheduler.jsx

```diff
@@ -5,6 +5,7 @@ import {contains} from 'ramda';
 import ReactDataGrid from 'react-data-grid';
 import ms from 'ms';
 import matchSorter from 'match-sorter';
+import * as prettyCron from 'prettycron';
```

Comment:
let's use something lighter that doesn't depend on `later`: https://www.npmjs.com/package/cronstrue